Src/Particle/AMReX_ParticleTile.H

```diff
@@ -720,6 +720,16 @@ struct ParticleTile
 
     ParticleTile () = default;
 
+#ifndef _WIN32  // workaround windows compiler bug
```

Comment:
@atmyers @WeiqunZhang This should probably read MSVC.

There are many compilers on Windows (LLVM/Clang, GCC aka MinGW, Intel, etc.) that should work file :)

Reply:
Ok, I see that LLVM/Clang failed, too.